.github/workflows/run-tests.yml

```
@@ -14,13 +14,15 @@ jobs:
       matrix:
         os: [ubuntu-latest, windows-latest]
         php: [8.0]
-        laravel: [8.*, 9.*]
+        laravel: [8.*, 9.*, 10.*]
         stability: [prefer-lowest, prefer-stable]
         include:
           - laravel: 8.*
             testbench: ^6.6
           - laravel: 9.*
             testbench: 7.*
+          - laravel: 10.*
+            testbench: 8.*
 
```

We should exclude certain combinations of php and laravel. Something like this.

```suggestion
        exclude:
          - laravel: 8.*
            php: 8.2
          - laravel: 10.*
            php: 8.0
```